Cargo.toml

```diff
@@ -34,6 +34,12 @@ rdkafka = { version = "0.28.0", features = [ "cmake-build", "gssapi-vendored", "
 [target.'cfg(not(target_os = "macos"))'.dependencies]
 rdkafka = { version = "0.28.0", features = [ "cmake-build", "gssapi", "ssl" ] }
 
+[target.'cfg(target_os = "windows")'.dependencies]
+sasl2-sys = {version="0.1.20",features=["vendored","openssl-vendored"]}
+
+[build-dependencies]
```

Comment:
Based on these issues, it can be determined that the `force-engine` feature is required for windows and musl platforms
https://github.com/fede1024/rust-rdkafka/issues/572#issuecomment-1529316876
https://github.com/fede1024/rust-rdkafka/issues/446
https://github.com/alexcrichton/openssl-src-rs/pull/151